google-cloud-spanner/src/test/java/com/google/cloud/spanner/connection/StatementTimeoutTest.java

```diff
@@ -1073,8 +1135,9 @@ public void run() {
               WAIT_BEFORE_CANCEL,
               TimeUnit.MILLISECONDS);
 
-      expected.expect(SpannerExceptionMatcher.matchCode(ErrorCode.CANCELLED));
       connection.execute(Statement.of(SLOW_DDL));
```

Comment:
Add a `fail("Expected exception")` to prevent the test case from succeeding in case of no error.